code/unit_tests/missing_icons.dm

```diff
@@ -0,0 +1,54 @@
+/// Makes sure objects actually have icons that exist!
+/datum/unit_test/missing_icons
```

Comment:
this is iffy; we don't adhere to the same icon hierarchy as tg. will this even work?

(even if it does i don't particularly want to enforce putting all obj icons in a few folders really, we don't use the same hierarchy for a reason)